docs/sv_settings.html

```diff
@@ -59,6 +59,8 @@ <h3><img src="assistant.png" alt="Hjälpgivare"/> &nbsp;<a id="assistant-details
 
       <p>
         Använd formuläret för att ange hur <em>hjälptagarens</em> skärm blir <strong>avläst</strong>; du kan anpassa (i millisekunder) tiden mellan två avläsningar (eller "tick") liksom antal gråskalor i bilden.
+        Från och med Dayon! 15 kan sRGB-färger också användas som ett alternativ. I detta fall ignoreras antalet gråtoner.
```

Comment:
@lammel-hub may I ask you to have a brief look at the Swedish grammar please?

- sv_settings.html
- Babylon_sv.properties

Very appreciated, many thanks in advance!